lib/avo/resources/base.rb

```diff
@@ -461,8 +461,12 @@ def fill_record(record, params, extra_params: [], fields: nil)
 
         # Write the user configured extra params to the record
         if extra_params.present?
+          # Pick only the extra params
+          # params at this point are already permited, only need the keys to access them
+          extra_attributes = params.slice(*flatten_keys(extra_params))
```

Comment:
How about we `params` to `permitted_params` so it's clear to anyone who's reading this and wondering why we are slicing instead of permitting.